azure-pipelines.yml

```diff
@@ -108,6 +110,8 @@ jobs:
           SourcePath: 'packages/fluentui/docs/src/public'
           storage: fabricweb
 
+      - template: .devops/templates/cleanup.yml
```

Comment:
do we need to check change here and below? shouldn't checking in buildtestlint step sufficient?

Reply:
Maybe not totally necessary, but I think I'd rather keep it just to be safe.